packages/circuit-ui/components/BaseStyles/BaseStylesService.ts

```diff
@@ -43,6 +43,158 @@ export const createBaseStyles = ({
       url('${FONTS_BASE_URL}/aktiv-grotest-700.eot') format('embedded-opentype');
   }
 
+  :root {
+    /* Neutral backgrounds */
+    --cui-bg-normal: #ffffff;
+    --cui-bg-normal-hovered: #f5f5f5;
+    --cui-bg-normal-pressed: #e6e6e6;
+    --cui-bg-normal-disabled: rgba(255, 255, 255, 0.4);
+    --cui-bg-subtle: #f5f5f5;
+    --cui-bg-subtle-hovered: #e6e6e6;
+    --cui-bg-subtle-pressed: #cccccc;
+    --cui-bg-subtle-disabled: rgba(245, 245, 245, 0.4);
+    --cui-bg-highlight: #e6e6e6;
+    --cui-bg-highlight-hovered: #cccccc;
+    --cui-bg-highlight-pressed: #999999;
+    --cui-bg-highlight-disabled: rgba(230, 230, 230, 0.4);
+    --cui-bg-strong: #000000;
+    --cui-bg-strong-hovered: #000000;
+    --cui-bg-strong-pressed: #000000;
```

Comment:
How will users perceive/differentiate these different states?

Reply:
In this case it looks like there is no differentiation. This could for instance be used in the ODL black buttons. From a standards perspective (WCAG 2.1 AA), nothing states that hovered or pressed states should be visually different from a UI element's idle state.

So we fall back to a design question—I'm happy to raise this with design again in the context of ODL (which introduces a lot of these black and white styles) but in the meantime I'd suggest moving forward here (no blocker and this is a design decision).

Reply:
I'm surprised this isn't an accessibility failure. I would at least classify it as a usability issue. Do you know if there's any other indicator planned in the new button design? 

I agree that this shouldn't block this PR, but I'd like to question this decision and improve it in the future.